web/src/components/profile/ProfileNewJerseyAddress.tsx

```diff
@@ -52,87 +56,97 @@ export const ProfileNewJerseyAddress = (): ReactElement => {
     setIsValidZipCode(!doesFieldHaveError("addressZipCode"));
   };
 
+  const areAddressFieldsDisabled = (): boolean => {
+    return business && isStartingBusiness(business) && business.formationData.completedFilingPayment
```

Comment:
pebble - the only value we care about is `business.formationData.completedFilingPayment` since oscars never form, we could remove `business && isStartingBusiness(business) &&`